docs/pkglists/Fedora_31.x86_64

```diff
@@ -46,6 +46,7 @@ x86_64/bareos-filedaemon-ldap-python-plugin
 x86_64/bareos-filedaemon-libcloud-python-plugin
 x86_64/bareos-filedaemon-ovirt-python-plugin
 x86_64/bareos-filedaemon-percona-xtrabackup-python-plugin
+x86_64/bareos-filedaemon-mariabackup-python-plugin
```

Comment:
The whole file shouldn't be part of the source tree !

Reply:
What do you mean? We added these files to make sure no package is lost accidentially (which already happened)